upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -61,232 +65,3131 @@ data:
 ---
 # Source: calico/templates/kdd-crds.yaml
 
-apiVersion: apiextensions.k8s.io/v1beta1
+
+---
```

Comment:
Empty resources have caused us problems in the past. Are we stripping all these out now?

Reply:
We are stripping them now. I thought is best to keep them as close as possible to the official manifest, even if not as clean.